build/images/base-windows/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM mcr.microsoft.com/windows/servercore:1809 as windows-utility-base
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV CNI_PLUGINS="./host-local.exe"
+
+WORKDIR /
+
+RUN mkdir -Force C:\opt\cni\bin
+
+RUN curl.exe -LO https://github.com/containernetworking/plugins/releases/download/v0.8.1/cni-plugins-windows-amd64-v0.8.1.tgz; \
+    tar -xzf cni-plugins-windows-amd64-v0.8.1.tgz  -C C:\opt\cni\bin ${env:CNI_PLUGINS}; \
+    rm cni-plugins-windows-amd64-v0.8.1.tgz
```

Comment:
would it be possible to use the version stored in `build/images/deps/cni-binaries-version` for this, or is it not possible to use the same version for Linux & Windows?